albis-leasing-sdk/src/helpers.js

```diff
@@ -60,41 +62,34 @@ export async function login(
   password,
   realm,
   audience,
-  grant_type,
+  grantType,
+  nodeEnv
 ) {
-  if (process.env.NODE_ENV === 'test')
-    return Promise.resolve({ data: { access_token: 'testAlbisToken12345' } });
+  if (nodeEnv === 'test')
+    return Promise.resolve({ data: { access_token: 'testAuth0Token12345' } });
   return axios.post(auth0Endpoint, {
     headers: { 'content-type': 'application/json' },
-    // username,
-    // password,
-    // realm,
+    username,
+    password,
+    realm,
     client_id: APIid,
     client_secret: APIsecret,
-    audience: audience,
-    grant_type: 'client_credentials',
+    audience,
+    grant_type: grantType,
   });
 }
 
-export function getEndpointPath(resource, devMode) {
-  if (resource === 'ping' && devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/staging/ping';
-  if (resource === 'ping' && !devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/v1/ping';
-  if (resource === 'rate' && process.env.NODE_ENV === 'test')
-    return 'http://localhost:3000/testModels/rates.json';
-  if (resource === 'rate' && devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/staging/rate';
-  if (resource === 'rate' && !devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/v1/rate';
-  if (resource === 'application' && devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/staging/application';
-  if (resource === 'application' && !devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/v1/application';
-  if (resource === 'legalForms' && devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/staging/legal-forms';
-  if (resource === 'legalForms' && !devMode)
-    return 'https://mzgz6czl6h.execute-api.eu-central-1.amazonaws.com/v1/legal-forms';
+export function getEndpointPath(resource, apiStage, SDKendpoint, nodeEnv) {
+  if (resource === 'ping')
+    return `${SDKendpoint}/${apiStage}/ping`;
+  if (resource === 'rate' && nodeEnv === 'test')
+    return `http://localhost:3000/testModels/rates.json`;
+  if (resource === 'rate')
+    return `${SDKendpoint}/${apiStage}/rate`;
+  if (resource === 'application')
+    return `${SDKendpoint}/${apiStage}/application`;
+  if (resource === 'legalForms')
+    return `${SDKendpoint}/${apiStage}/legal-forms`;
```

Comment:
Remove all if statement and make one, like:
return `${SDKendpoint}/${apiStage}/${resource}`